deploy/docker/cp-dts-status-checker/dts-status-checker.py

```diff
@@ -0,0 +1,139 @@
+from functools import total_ordering
+import json
+import subprocess
+import os
+import sys
+from pipeline.api import PipelineAPI
+from pipeline.log.logger import LocalLogger, TaskLogger, RunLogger
+import codecs
+import copy
+
+FS_ROOT = "/"
+CLOUD_DATA_DIR = "cloud-data"
+DTS_STATUS_FILE = "data.json"
+DTS_EMAIL_TEMPLATE = "template.html"
+
+dts_user = os.environ.get('CP_DTS_STATUS_NOTIFICATION_USER')
+if dts_user is None:
```

Comment:
in python it is better to check for strings as:
```
if string:
   <do something>
```
because it will check that string is not None and also that it is not empty